app/assets/src/components/views/SampleViewV2/SampleViewV2.jsx

```diff
@@ -60,7 +69,8 @@ export default class SampleViewV2 extends React.Component {
         view: "table",
         selectedOptions: this.defaultSelectedOptions(),
       },
-      localState
+      localState,
+      urlState
```

Comment:
This part feels hard to read. It's hard to know which values are stored in localState and which ones are stored in urlState (looks like there's a lot of overlap). It looks like you have to cross-reference multiple functions to get the full list of fields stored in each. Is there any way to organize all that info in one place for better readability?

Reply:
Did the change with fields reference in constants at the top.